src/db/postgres.py

```diff
@@ -410,7 +417,23 @@ def parse_plan(self):
 
             for prop in node_props[1:]:
                 if prop.startswith(' '):
-                    node.properties.append(prop.strip())
+                    prop_str = prop.strip()
+                    if match := pat_decompose_hash_properties.search(prop_str):
+                        node.properties['Hash Buckets'] = match.group('buckets')
+
+                        if orig_buckets := match.group('orig_buckets'):
+                            node.properties['Original Hash Buckets'] = orig_buckets
+
+                        node.properties['Hash Batches'] = match.group('batches')
+
+                        if orig_batches := match.group('orig_batches'):
+                            node.properties['Original Hash Batches'] = orig_batches
+
+                        node.properties['Peak Memory Usage'] = match.group('peak_mem')
+                    else:
+                        keylen = prop_str.find(':')
+                        if keylen > 0:
+                            node.properties[prop_str[0:keylen]] = prop_str[keylen+1:].strip()
```

Comment:
`node.properties[prop_str[0:keylen]]` can be w/o 0 ` node.properties[prop_str[:keylen]]`

Reply:
Oh, yes, nice catch.